src/pint/pintk/pulsar.py

```diff
@@ -388,6 +388,8 @@ def add_jump(self, selected):
             )
             if self.fitted:
                 self.postfit_model.add_component(a)
+            log.info(f"New jump {retval} added for {selected.sum()} toas.")
```

Comment:
So I  just noticed that this is now a `log.info` *and* a `print`.  I think that we were suggesting that if anything changes with the JUMPs that you should give a `log.info` *rather than* a `print`.  I just noticed another `print` up above, as well, that should probably be a `log.info`.